pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -144,6 +145,92 @@ func (r *ReconcileComplianceScan) Reconcile(request reconcile.Request) (reconcil
 	return reconcile.Result{}, nil
 }
 
+func (r *ReconcileComplianceScan) handleRootCASecret() error {
+	secret := &corev1.Secret{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{Name: "compliance-operator-root-ca", Namespace: "openshift-compliance"}, secret)
+	if err == nil {
+		// Already exists, so return.
+		return nil
+	}
+	if !errors.IsNotFound(err) {
+		return err
+	}
+
+	// Create the CA.
+	cert, key, err := utils.ComplianceOperatorRootCA("compliance-operator-root-ca", 100)
+	if err != nil {
+		return err
+	}
+
+	secret = &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "compliance-operator-root-ca",
+			Namespace: "openshift-compliance",
```

Comment:
I know the namespace is probably not going to change, ever, but do you think it should be either defined globally or passed from the scan instance?

Reply:
I'd dig a global variable or a function.

... we already have a function for this: https://github.com/openshift/compliance-operator/blob/master/cmd/manager/main.go#L165

Reply:
Addressed this